backend/app/crud/credentials.py

```
@@ -107,6 +107,26 @@
 
 
+def get_full_provider_credential(
+    *, session: Session, org_id: int, provider: str, project_id: Optional[int] = None
+) -> Optional[Credential]:
+    """Fetches credentials for a specific provider of an organization."""
+    validate_provider(provider)
+
+    statement = select(Credential).where(
+        Credential.organization_id == org_id,
+        Credential.provider == provider,
+        Credential.is_active == True,
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if Credential.is_active:` for truth checks

Replace with `Credential.is_active`

(E712)